### PR TITLE
Fix navbar

### PR DESCRIPTION
**Pull request by anishpawaskar**
**Closes:** https://github.com/Real-Dev-Squad/members-site/issues/104

**What is change?**
- Remove crypto button from navbar
- Made RDS logo clickable and when user click on it they will be redirected to the home page

Backend changes:
- [ ] Yes
- [x] No

Frontend Changes:
- [x] Yes
- [ ] No

Database changes:
- [ ] Yes
- [x] No

Breaking changes (If your feature is breaking/missing something please mention pending tickets)
- [ ] Yes
- [x] No

**Before:**

https://github.com/Real-Dev-Squad/members-site/assets/22213872/81e02dc1-dadb-4980-8a8f-655efc778b00

**After:**

https://github.com/Real-Dev-Squad/members-site/assets/22213872/9354df2f-f0f2-468a-ac5b-91075181babb

### CODE REVIEW

**Comment by ajoykumardas12** [comment]
Please add test stats. 

> Closes: https://github.com/Real-Dev-Squad/members-site/issues/104

Also, does this PR close #104 ?  #104 has more tasks.

**Comment by anishpawaskar** [comment]
> Please add test stats.
> 
> > Closes: #104
> 
> Also, does this PR close #104 ? #104 has more tasks.

Test are not written i have mentioned in design doc that test will be covered in second milestone where im creating dropdown component and yes it closes #104 but not now

File: src/components/Layout/Navbar/NavbarDesktop.tsx [hunk 1]
```diff
@@ -45,23 +37,17 @@ export default function NavbarDesktop() {
...
-        display: "flex",
-        padding: "20px",
-        margin: 0,
-        gap: "20px",
-        alignItems: "center",
-      }}
+      className={styles.navbarMenu}
     >
       <ListItem>
+        <Link href={HOME_URL}>
```
Comment (ajoykumardas12) [inline src/components/Layout/Navbar/NavbarDesktop.tsx:43]: Not sure if Link component has any benefit here, the link isn't pointing to another route, rather different url.

File: src/components/Layout/Navbar/NavbarDesktop.tsx [hunk 2]
```diff
@@ -45,23 +37,17 @@ export default function NavbarDesktop() {
...
         <Image
           src="/images/Real-Dev-Squad@1x.svg"
-          width={50}
-          height={50}
+          width={45}
+          height={45}
```
Comment (ajoykumardas12) [inline src/components/Layout/Navbar/NavbarDesktop.tsx:47]: Hope the size changes has been discussed, didn't see it mentioned in issue.

File: src/components/Layout/Navbar/navbar.module.css [hunk 3]
```diff
@@ -3,4 +3,25 @@
     top: 0;
     width:100%;
     z-index:1;
-}
\ No newline at end of file
+}
+
+.navbarMenu {
```
Comment (ajoykumardas12) [inline src/components/Layout/Navbar/navbar.module.css:8]: nit: saw snake case is followed mostly in the styles of members-site.

File: src/components/Layout/Navbar/navbar.module.css [hunk 4]
```diff
@@ -3,4 +3,25 @@
...
+    padding: 20px;
+    margin: 0;
+    gap: 3.4rem;
+    align-items: center;
+    padding-left: 40px;
+}
+
+.navlinks {
+    font-weight: 700;
+    font-size: 16px;
```
Comment (ajoykumardas12) [inline src/components/Layout/Navbar/navbar.module.css:21]: Font size should be in rem unit.

File: src/components/Layout/Navbar/NavbarDesktop.tsx (hunk 2 above)
Comment (anishpawaskar) [reply src/components/Layout/Navbar/NavbarDesktop.tsx:47]: There is a difference  between status site navbar and member site navbar so to bring the consistency in navbar so i have set the size of logo according to status site

File: src/components/Layout/Navbar/navbar.module.css [hunk 5]
```diff
@@ -3,4 +3,25 @@
...
+}
+
+.navlinks:hover {
+    color: #49a82e;
```
[the remark this answers is not in the document]
Comment (anishpawaskar) [reply src/components/Layout/Navbar/navbar.module.css:25]: This color is being used only once so we should keep it as it is

**Review by vinayak-trivedi — APPROVED** [review]
聖川真斗食べる　たこ焼きGIF

File: src/components/Layout/Navbar/navbar.module.css (hunk 4 above)
Comment (anishpawaskar) [reply src/components/Layout/Navbar/navbar.module.css:21]: Okay i will change it

**Pull request merged**